test/email_validator_test.dart

```diff
@@ -116,15 +116,10 @@ void main() {
     'θσερ@εχαμπλε.ψομ', // Greek
   ];
 
-  test('Validate that validate throws error on null email', () {
-    expect(() => EmailValidator.validate(null),
-        throwsA(new isInstanceOf<ArgumentError>()));
-  });
-
   test('Validate invalidAddresses are invalid emails', () {
     invalidAddresses.forEach((String actual) => expect(
```

Comment:
(Dirve-by nit: Consider using `for (var actual in invalidAddresses) { ... }` instead of `forEach`. https://dart.dev/guides/language/effective-dart/usage#avoid-using-iterableforeach-with-a-function-literal)

Reply:
Yeah this is quite old, will fix. Thanks.